### ISSUE TITLE
add to cart

### ISSUE THREAD

**Issue by badgeek** [lovata/oc-shopaholic-plugin#63]
possible to have add to cart function (not replacing an item, but add more quantity )

**Comment by shustrya** [comment]
Is it about adding goods to a basket with extended properties?

--
Было бы здорово передавать вместе с "Товарным предложением" и "Количеством" еще пару произвольных свойств из формы товара, и видеть их в заказах.

**Comment by kharanenka** [comment]
No, in this issue we are talking about adding the methods of increase() / decrease(). When the offers are in the cart and you can to increase / decrease the quantity of the offers.

Please, create a new issue with a description and an example when you need to send additional properties in order from product form.
